.github/workflows/on_pull_request.yml

```diff
@@ -418,6 +418,11 @@ jobs:
         run: yarn test:website
         env:
           CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+          APPLITOOLS_API_KEY: ${{ secrets.APPLITOOLS_API_KEY }}
```

Comment:
@SiTaggart @TheSisb I couldn't find a record of this API key being removed. Can either of you confirm that it is still available on this `secrets` object?

Reply:
You should be able to view what secrets are set under the repo settings in GitHub. I don't think access to that is restricted by role if you're a maintainer of the repo, but I could be wrong

Reply:
Yeah I don't have access.

Reply:
OK, well rest assured the secret for applitools is still there